deployment/terraform/cloud-config/ecs-batch-user-data.yml

```diff
@@ -0,0 +1,78 @@
+#cloud-config
+
+packages:
+    - awslogs
+
+fs_setup:
```

Comment:
From http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html:

[NVMe instance store volumes] These volumes are mapped automatically as /dev/nvme[0-7]n1. You do not need to specify the NVMe volumes supported by an instance type in a block device mapping.

All this does is make the block device visible to the OS, e.g. via `fdisk`. You still need to mount it yourself, which is the purpose of the additions in the cloud-config file.